src/Paybox.php

```diff
@@ -464,17 +140,24 @@ private function configurePayboxCallParameters($operation, OptionsResolver $reso
         $resolver
             ->setAllowedValuesIfDefined('ACQUEREUR', ['PAYPAL', 'EMS', 'ATOSBE', 'BCMC', 'PSC', 'FINAREF', 'BUYSTER', '34ONEY'])
             ->setAllowedValuesIfDefined('ACTIVITE', [
-                PayboxVariableActivity::NOT_SPECIFIED,
-                PayboxVariableActivity::PHONE_REQUEST,
-                PayboxVariableActivity::MAIL_REQUEST,
-                PayboxVariableActivity::MINITEL_REQUEST,
-                PayboxVariableActivity::WEB_REQUEST,
-                PayboxVariableActivity::RECURRING_PAYMENT,
+                Activity::NOT_SPECIFIED,
+                Activity::PHONE_REQUEST,
+                Activity::MAIL_REQUEST,
+                Activity::MINITEL_REQUEST,
+                Activity::WEB_REQUEST,
+                Activity::RECURRING_PAYMENT,
+            ])
+            ->setAllowedValuesIfDefined('DEVISE', [
+                null,
+                Currency::EURO,
+                Currency::US_DOLLAR,
+                Currency::CFA,
```

Comment:
I know you're not a big fan of external dependencies, but shameless plug : https://github.com/greg0ire/enum . Might help you replace the `ALL` const with something shorter. And there is a bundle if you're interested : https://github.com/greg0ire/enum-bundle